image/base/install-nginx

```diff
@@ -2,13 +2,13 @@
 set -e
 
 # version check: https://nginx.org/en/download.html
-VERSION=1.25.3
-HASH="64c5b975ca287939e828303fa857d22f142b251f17808dfe41733512d9cded86"
+VERSION=1.26.1
 
 cd /tmp
 wget -q https://nginx.org/download/nginx-$VERSION.tar.gz
-sha256sum nginx-$VERSION.tar.gz
-echo "$HASH nginx-$VERSION.tar.gz" | sha256sum -c
+wget -q https://nginx.org/download/nginx-$VERSION.tar.gz.asc
+gpg --import nginx-$VERSION.tar.gz.asc
```

Comment:
`gpg --import` will import one or more PGP public keys into the GnuPG key ring.  This `.asc` file is not a public key -- it is a signature.

```
% gpg --import nginx-1.26.1.tar.gz.asc
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
% echo $?
2
```

Release keys are published here.
https://nginx.org/en/pgp_keys.html

```
% curl -Ssf https://nginx.org/keys/nginx_signing.key | gpg --import
gpg: key 2FD21310B49F6B46: 4 signatures not checked due to missing keys
gpg: key 2FD21310B49F6B46: public key "nginx signing key <signing-key-2@nginx.com>" imported
gpg: key ABF5BD827BD9BF62: 6 signatures not checked due to missing keys
gpg: key ABF5BD827BD9BF62: public key "nginx signing key <signing-key@nginx.com>" imported
gpg: key BCDCD8A38D88A2B3: 3 signatures not checked due to missing keys
gpg: key BCDCD8A38D88A2B3: public key "nginx signing key <signing-key-3@nginx.com>" imported
gpg: Total number processed: 3
gpg:               imported: 3
gpg: no ultimately trusted keys found
% echo $?
0
```

The line below this one makes sense.  Once the release keys are imported, you may verify the signature in the `.asc` file.

Reply:
Thank you for the tip here 👍 I've applied your feedback into the PR.